ghost/webmentions/package.json

```diff
@@ -20,7 +20,11 @@
   "devDependencies": {
     "c8": "7.12.0",
     "mocha": "10.2.0",
+    "nock": "^13.3.0",
     "sinon": "15.0.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "cheerio": "^1.0.0-rc.12",
+    "node-fetch": "2"
+  }
```

Comment:
For dependencies we always pin them to a fixed number, and keep those numbers the same across all the packages. So usually I search for `cheerio":` in code to find the current version we use for `cheerio` in a different package, and then set that version in `package.json`.

In this case, nock should be `13.3.0` for example.